test/integration/single-list.spec.js

```diff
@@ -0,0 +1,60 @@
+// @flow
```

Comment:
Given we are starting a browser in this test, I think it best to add the node pragma to the top of this file:

```js
/**
* @jest-environment node
*/
// @flow
```

That way we can be sure we are not touching jsdom

Reply:
@TrySound - this was my original approach but after discussing with @alexreardon  and as we have only 1 spec file for now when we will have more than 1 spec it makes more sense to have  jest-environment-puppeteer.